instrumentation/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmq/RockerMqConsumerAttributeExtractor.java

```diff
@@ -24,7 +23,7 @@ protected String system(MessageExt request) {
 
   @Override
   protected String destinationKind(MessageExt request) {
-    return SemanticAttributes.MessagingDestinationKindValues.TOPIC;
+    return "group";
```

Comment:
This method is supposed to return values defined in specification, which are `topic` and `queue`. See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md If you want to introduce new values you should first change the specification.

Reply:
You mean to add group to constant class?

Reply:
To add a new constant you'll have to add it to specification. I'd suggest you to consider whether you really need to change this.